src/main/java/uk/gov/hmcts/reform/sscs/model/NotificationEventType.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+public enum NotificationEventType {
+
+    ADJOURNED_NOTIFICATION("hearingAdjourned"),
```

Comment:
we shouldn't put all the events in here but only the one that is needed, really.

Reply:
Agree